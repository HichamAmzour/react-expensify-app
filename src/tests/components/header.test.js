import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../Components/Header';
// import toJson from 'enzyme-to-json';


import {shallow} from 'enzyme';
test("Should reneder Header component correctly",()=>{
    const wrapper= shallow(<Header/>)
    expect(wrapper).toMatchSnapshot()
    // const wrapper= shallow(<Header/>)
    // expect(wrapper.find("h1").text()).toBe("this is the Header Expensify")
})