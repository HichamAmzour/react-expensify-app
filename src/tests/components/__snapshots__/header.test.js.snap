// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should reneder Header component correctly 1`] = `
<header>
  <h1>
    this is the Header Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help 
  </NavLink>
</header>
`;
